app/models/submitCreateMovement/ImportCustomsDeclarationModel.scala

```diff
@@ -24,11 +24,11 @@ import play.api.libs.json.{Json, OFormat}
 import queries.SadCount
 import utils.{Logging, ModelConstructorHelpers}
 
-case class ImportSadModel(importSadNumber: String)
+case class ImportCustomsDeclarationModel(importCustomsDeclarationNumber: String)
 
-object ImportSadModel extends ModelConstructorHelpers with Logging {
+object ImportCustomsDeclarationModel extends ModelConstructorHelpers with Logging {
 
-  def apply(implicit request: DataRequest[_]): Seq[ImportSadModel] = {
+  def apply(implicit request: DataRequest[_]): Seq[ImportCustomsDeclarationModel] = {
     request.userAnswers.getCount(SadCount) match {
       case Some(0) | None =>
         logger.error("SadSection should contain at least one item")
```

Comment:
Updated log message would helpful

Reply:
As mentioned before, 'sad' is not changed everywhere. Here SadSection is part of `UserAnswers` from where we try to get the sadNumber and linked it to CustomDeclaration number.